apps/src/applab/RedirectDialog.jsx

```diff
@@ -69,12 +67,24 @@ export default class RedirectDialog extends React.Component {
     }
 
     return (
-      <Dialog title={title} isOpen={this.state.isOpen}>
-        <Body>
+      <Dialog title={title} isOpen handleClose={this.props.handleClose}>
```

Comment:
isOpen <- supposed to be removed? Or should it be set equal to something?

Reply:
Originally, the isOpen state was used to determine whether the dialog was to be open or closed. Now we track the number of redirects and if there are no redirects, return null. So if the code gets to this line, redirects do exist and we want to display this dialog, so isOpen was set to permanently true.